src/platforms/common/data-management/sensitive-data/attachment-scrubbing.mdx

```diff
@@ -0,0 +1,92 @@
+- *Hash*: Hashing happens as usual, however the **binary representation** of the match is hashed, regardless of the encoding the match was made in.  The resulting hash is padded or truncated as described above.
+- *Replace*: This behaves like normal, with the replacement being padded or truncated as described above.
+
+## Source selectors
+
+All attachments can be selected using the `$attachments` [Value Type](../advanced-datascrubbing/#value-types) selector at the root of the selector, this can be followed by the specific filename itself.
+The filename part needs to be quoted to write the actual filename but wildcards can be used as well.
+
+For example if your filename is `minidump.dmp` you can select all fields which can be scrubbed using `$attachments.'minidump.dmp'.**`.
+To write the same using a wildcard to select all attachments you would write `$attachmetns.*.**` or even simply `$attachments.**`.
+
+### Minidump selectors
+
+A minidump is always an attachment, but it also has its own [value type](../advanced-datascrubbing/#value-types) of `$minidump`.
+This allows you to select all fields which can be scrubbed in a minidump using `$minidump.**`.
+This really is a shorthand for selecting the fields of all attachments which are minidumps: `$attachments.$minidump.**`.
+
+So far we have repeatedly referred to all fields in a minidump which can be scrubbed.
+For the puprpose of PII scrubbing we currently represent the minidump as a flat structure with the following fields:
+
+- The **stack memory**: These are memory regions in the minidump which are used as stack memory by a thread.
+  The [value type](../advanced-datascrubbing/#value-types) selector `$stack_memory` can be used to select these regions.
+  Generally they will contain binary data so be careful with the [data types](../advanced-datascrubbing/#data-types) used to scrub these regions.
+
+  In particular for stack memory it is **not recommended** to be scrubbed, since this is used by the event processing pipeline to unwind the stack and build a readable stack trace for the event.
+  Inadvertedly modifying the stack in an undesired way will make this process impossible and the event much less useful.
+
+  In general all [data types](../advanced-datascrubbing/#data-types) result in a textual regular expression match on some data, for the stack this regular expression is matched as a binary UTF-8 regular expression so that any UTF-8 embedded in the binary data will be matched.
+  Additionally the regular expression is applied to any valid UTF-16LE strings found within the binary data.
+
+- The **heap memory**: For the current implementation this is considered to be any memory region which is not used as stack memory.
+  This means it could include other non-stack regions like `mmap`ed files.  All these memory regions can be selected using the `$heap_memory` [value type](../advanced-datascrubbing/#value-types) selector.
```

Comment:
I'm somewhat reluctant to simple replace "it" with "heap memory" as this is kind of not wholly correct since this paragraph is trying to explain that the "heap memory" name is a bit of a misnomer.  Can we use "This means these memory regions could include ..."?  Or is that too ambiguous again?

"such as `mmap` fiules" is unfortunately incorrect.  Would "such as files which are mapped to the process' memory using `mmap`." be a good way to finish that sentence?

Reply:
Generally, the term "this" should be followed by a noun for greater clarity, but it's not a strict requirement. I think both of the changes you suggest are fine, with one minor edit - "such as files *that* are mapped to the process' memory using `mmap`."